docs/sections/papers/ultrafeedback.md

```diff
@@ -46,43 +47,66 @@ from distilabel.steps import (
 )
 from distilabel.steps.tasks import TextGeneration, UltraFeedback
 
+sample_three_llms = sample_n_steps(3)
+
 
 with Pipeline(name="ultrafeedback-pipeline") as pipeline:
     load_hub_dataset = LoadHubDataset(
         name="load_dataset",
         output_mappings={"prompt": "instruction"},
+        batch_size=2,
     )
 
     text_generation_with_notus = TextGeneration(
         name="text_generation_with_notus",
         llm=vLLM(model="argilla/notus-7b-v1"),
-        input_batch_size=10,
+        input_batch_size=2,
         output_mappings={"model_name": "generation_model"},
     )
     text_generation_with_zephyr = TextGeneration(
         name="text_generation_with_zephyr",
         llm=vLLM(model="HuggingFaceH4/zephyr-7b-gemma-v0.1"),
-        input_batch_size=10,
+        input_batch_size=2,
+        output_mappings={"model_name": "generation_model"},
+    )
+    text_generation_with_gemma = TextGeneration(
+        name="text_generation_with_gemma",
+        llm=vLLM(model="google/gemma-1.1-7b-it"),
+        input_batch_size=2,
+        output_mappings={"model_name": "generation_model"},
+    )
+    text_generation_with_zephyr_gemma = TextGeneration(
+        name="text_generation_with_zephyr_gemma",
+        llm=vLLM(model="HuggingFaceH4/zephyr-7b-gemma-v0.1"),
+        input_batch_size=2,
+        output_mappings={"model_name": "generation_model"},
+    )
+    text_generation_with_llama = TextGeneration(
+        name="text_generation_with_llama",
+        llm=vLLM(model="meta-llama/Meta-Llama-3-8B-Instruct"),
+        input_batch_size=2,
+        output_mappings={"model_name": "generation_model"},
+    )
+    text_generation_with_ultramerge = TextGeneration(
+        name="text_generation_with_ultramerge",
+        llm=vLLM(model="mlabonne/UltraMerge-7B"),
+        input_batch_size=2,
         output_mappings={"model_name": "generation_model"},
     )
-    load_hub_dataset.connect(text_generation_with_notus)
-    load_hub_dataset.connect(text_generation_with_zephyr)
 
     combine_columns = CombineColumns(
         name="combine_columns",
         columns=["generation", "generation_model"],
         output_columns=["generations", "generation_models"],
+        input_batch_size=2
```

Comment:
This shouldn't be required right?

Reply:
It is. By default the `input_batch_size` is 50, but as this is a convergence step (it will receive batches of steps that received routed batches) it has to be lower or equal than the `input_batch_size` of the previous steps.

Reply:
Then does that mean that if in one step we define `input_batch_size=1` we are forcing the rest of the steps below in the pipeline to use the same batch size?